### PR TITLE
feat: Google Cloud Platform page added

### PR DESCRIPTION
**Pull request by nabinelnino**

### CODE REVIEW

File: docs/software_development/Remote_Development/Google_Cloud_Platform/index.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+### Prerequisites:
+
+- Ensure you have an active Gmail account.
+- Confirm that your account has been added to the relevant GCP project.
+
+#### To access and use Google Cloud Platform (GCP), follow these steps:
+
+#### Accessing GCP via the Console
+
+- Visit the [Google Cloud Console](#https://console.cloud.google.com/) .
```
Comment (caryn-geady) [inline docs/software_development/Remote_Development/Google_Cloud_Platform/index.md:39]: link doesn't work
```suggestion
- Visit the [Google Cloud Console](https://console.cloud.google.com/).
```

File: docs/software_development/Remote_Development/Google_Cloud_Platform/index.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+
+- Explore the dashboard to view, manage, and configure services, projects, and resources.
+
+#### Accessing GCP via Terminal
+
+**To interact with GCP directly from your terminal:**
+**Step 1**: Initialize Google Cloud SDK
+
+- Install the Google Cloud SDK on your machine by following the official installation guide (https://cloud.google.com/sdk/docs/install).
```
Comment (caryn-geady) [inline docs/software_development/Remote_Development/Google_Cloud_Platform/index.md:48]: link
```suggestion
- Install the Google Cloud SDK on your machine by following the official [installation guide] (https://cloud.google.com/sdk/docs/install).
```

File: docs/software_development/Remote_Development/Google_Cloud_Platform/index.md [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+   bq load --source_format=CSV <DATASET_NAME>.<TABLE_NAME> gs://<BUCKET_NAME>/<FILE_NAME>
+   ```
+2. **Query Data**:  
+   Use BigQuery's web interface or CLI to run SQL queries for data cleaning, feature engineering, and exploratory analysis.
+   Example:
+   ````
+   SELECT * FROM `project_id.dataset_name.table_name` LIMIT 10;```
+   ````
+
+**Follow the instructions in this [document](#https://cloud.google.com/bigquery/docs) to learn more about BigQuery**
```
Comment (caryn-geady) [inline docs/software_development/Remote_Development/Google_Cloud_Platform/index.md:160]: link
```suggestion
**Follow the instructions in this [document](https://cloud.google.com/bigquery/docs) to learn more about BigQuery**
```

File: docs/software_development/Remote_Development/Google_Cloud_Platform/index.md [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+
+1. **Managed Service**: Automated backups, updates, and maintenance.
+2. **Scalability**: Seamless scaling for growing workloads.
+3. **Security**: Built-in encryption, IAM-based access, and network security.
+4. **Integration**: Works seamlessly with GCP services like Compute Engine, Kubernetes Engine, and BigQuery.
+5. **Flexibility**: Supports popular relational databases: MySQL, PostgreSQL, and Microsoft SQL Server.
+
+### **Setting Up Cloud SQL**
+
+Ensure that you have completed [How to Use GCP](#3-how-to-use-gcp) before starting this process.\_
```
Comment (caryn-geady) [inline docs/software_development/Remote_Development/Google_Cloud_Platform/index.md:176]:
```suggestion
Ensure that you have completed [How to Use GCP](#3-how-to-use-gcp) before starting this process.\
```

File: docs/software_development/Remote_Development/Google_Cloud_Platform/index.md [hunk 5]
```diff
@@ -0,0 +1,322 @@
...
+gcloud sql databases create [DATABASE_NAME] --instance=[INSTANCE_NAME]
+```
+
+**Add a User**:
+
+```
+gcloud sql users create [USERNAME] --password=[PASSWORD] --instance=[INSTANCE_NAME]
+```
+
+**For a detailed guide on using client services, please refer to this [link](#https://cloud.google.com/sql/docs)**
```
Comment (caryn-geady) [inline docs/software_development/Remote_Development/Google_Cloud_Platform/index.md:220]: link
```suggestion
**For a detailed guide on using client services, please refer to this [link](https://cloud.google.com/sql/docs)**
```

**Review by caryn-geady — CHANGES_REQUESTED** [review]
looks great and very informative - just need to fix a few web links

**Comment by nabinelnino** [comment]
fixed the suggested changes

**Comment by jjjermiah** [comment]
is this good to go @nabinelnino @mtran-code ?

**Comment by mtran-code** [comment]
I pushed the latest edits so maybe a review from anyone else first and then it's good.

**Pull request merged**